models/movie.js

```diff
@@ -59,23 +51,21 @@ movieSchema.set('toJSON', {
  * Given a person ID, ensures that it references an existing person.
  *
  * If it's not the case or the ID is missing or not a valid object ID,
- * the "directorHref" property is invalidated instead of "director".
- * (That way, the client gets an error on "directorHref", which is the
- * property they sent, rather than "director", which they don't know.)
+ * the "directorId" property is invalidated.
  */
 function validateDirector(value) {
   return new Promise((resolve, reject) => {
 
     if (!value) {
-      throw new Error(`directorHref Path \`directorHref\` is required`);
+      throw new Error(`directorId is required`);
```

Comment:
There's no longer a reason to do this check by hand. Please remove it and replace it with a `required: true` validator on the schema property.